### ISSUE TITLE
Support php7?

### ISSUE THREAD

**Issue by flyinghail** [c9s/php-r3#2]
HI.
PHP 7.0.0 Already Released http://php.net/archive/2015.php#id2015-12-03-1 .
Do you plan to add support to the PHP 7 in a near future?

**Comment by c9s** [comment]
Yes. this project is actually  in experimental stage. however any PR for supporting php7 is welcomed.

**Comment by patrickkh7788** [comment]
3 years passed, maybe put some message on read me if not plan to maintain.
